### PR TITLE
Adjust cluster-essentials reconciler logging

### PR DESCRIPTION
**Pull request by moelsayed**

### CODE REVIEW

**Review by pPrecel — APPROVED** [review]

File: pkg/reconciler/instances/clusteressentials/clusteressentials.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ const ReconcilerName = "cluster-essentials"
 func init() {
 	log := logger.NewLogger(false)
 
-	log.Debugf("Initializing component reconciler '%s'", ReconcilerName)
+	log.Infof("Initializing component reconciler '%s'", ReconcilerName)
```
Comment (jakobmoellersap) [inline pkg/reconciler/instances/clusteressentials/clusteressentials.go:14]: Is this intended to be logged at higher level now? Just checking for safety.

**Review by jakobmoellersap — APPROVED** [review]
If

File: pkg/reconciler/instances/clusteressentials/clusteressentials.go (hunk 1 above)
Comment (moelsayed) [reply pkg/reconciler/instances/clusteressentials/clusteressentials.go:14]: Yes. I think it makes sense to at least log the activation of the component reconciler. This provides feedback without too much noise. Making the reconciler completely silent in production could make it hard to know if it's even working or not.

**Pull request merged**